components/Navbar.js

```diff
@@ -18,12 +18,9 @@ const Navbar = () => {
           </span>
         </a>
         <div class="flex md:order-2">
-          <button
-            type="button"
-            class="bg-orange-400 rounded-lg text-sm text-center"
-          >
+          <li className="block py-2 pl-3 pr-4">
             {publicKey ? <WalletDisconnectButton /> : <WalletMultiButton />}
```

Comment:
![image](https://user-images.githubusercontent.com/28806828/211303401-0eea85db-d9d2-4bc8-91cd-29be05b65d4f.png)
The wallet connect button is invisible now. 
Please add `rounded-md and bg-black` to the li tag